Neo4j.Driver/Neo4j.Driver.IntegrationTests/RoutingDriver/RoutingDriverIT.cs

```diff
@@ -124,8 +128,8 @@ public void ShouldDisallowMoreStatementAfterDriverDispose()
 
             driver.Dispose();
             var error = Record.Exception(() => session.Run("RETURN 1"));
-            error.Should().BeOfType<ClientException>();
-            error.Message.Should().Contain("The current session cannot be reused as the underlying connection with the server has been closed");
+            error.Should().BeOfType<ObjectDisposedException>();
+            error.Message.Should().StartWith("Cannot acquire a new connection as LoadBalancer has already been disposed.");
```

Comment:
Would be nice to signal with this message that driver is disposed. "LoadBalancer" could be confusing for users.